driver/normalizer/normalizer.go

```diff
@@ -167,9 +167,8 @@ func (op opArrToChain) Construct(st *State, n nodes.Node) (nodes.Node, error) {
 	return typ, nil
 }
 
-func funcDefMap(typ string, returns bool, other Obj) Mapping {
+func funcDefMap(typ string, returns bool, other Obj, toGroup ...Op) Mapping {
```

Comment:
I don't entirely get what's going on here with `toGroup`. It seems like this is used to detect "collapsable" grouping, but I don't follow why it's _here_, and also why it's (potentially) _repeated_.

Could you add a comment to sketch (at least at a high level) what this is doing?

Reply:
Sorry, I am still having trouble following. Are the other fields you mentioned here populated from the underlying syntax we parsed, or are these values being injected into the transformed tree as part of the transform?